mass2/core/channel.py

```diff
@@ -45,6 +45,7 @@ class Channel:
     df: pl.DataFrame = field(repr=False)
     header: ChannelHeader = field(repr=True)
     npulses: int
+    subframediv: int = 64
```

Comment:
What is the value of having a default here? We could default to None and throw an error on any methods that need it. Or we could require it, and fix all uses. I don't think having a default of 64 is good, it opens up being silently wrong.